test/config/200-clusterrole.yaml

```diff
@@ -16,6 +16,11 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: knative-testing-admin
+  labels:
+    # This label includes this into the capabilities of the
```

Comment:
The comment doesn't compile for me.

Reply:
Added a link to: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles